tests/ztimer_xsec/Makefile

```diff
@@ -4,6 +4,9 @@ USEMODULE += ztimer
 USEMODULE += ztimer_usec
 USEMODULE += ztimer_msec
 USEMODULE += ztimer_sec
+USEMODULE += ztimer_periph_lptimer
+
+CFLAGS += "-DCONFIG_ZTIMER_LPTIMER_DEV=TIMER_DEV(1)" "-DCONFIG_ZTIMER_LPTIMER_FREQ=1000LU" "-DCONFIG_ZTIMER_LPTIMER_WIDTH=32" "-DCONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE=ZTIMER_CLOCK_NO_REQUIRED_PM_MODE"
```

Comment:
This will break for all non-stm32 boards

Reply:
i will remove that -- maybe i should remove the whole example commit ?

Reply:
Yea it's probably for the better